pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <parent>
+    <groupId>com.celements</groupId>
+    <artifactId>celements</artifactId>
+    <version>3.3-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.celements</groupId>
+  <artifactId>celements-cleverreach</artifactId>
+  <version>3.0-SNAPSHOT</version>
+  <description>Celements connector to Clever Reach API</description>
+  <dependencies>
+    <dependency>
+      <groupId>com.celements</groupId>
+      <artifactId>celements-xwiki-core</artifactId>
+      <version>3.5</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.celements</groupId>
+      <artifactId>celements-model</artifactId>
+      <version>3.11</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.celements</groupId>
+      <artifactId>celements-core</artifactId>
+      <version>3.7</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.glassfish.jersey.core</groupId>
+      <artifactId>jersey-client</artifactId>
+      <!-- Java 8 required before including 2.26+ -->
+      <version>2.25.1</version>
```

Comment:
different version than the one we already include (2.22.1). may we update `jersey-client` to 2.25.1 in general?

Reply:
where? ... I can't find anything in the pom files.

Reply:
it's included for jasper in the reporting repo